webapp/stores/notification_store.jsx

```diff
@@ -63,7 +63,9 @@ class NotificationStoreClass extends EventEmitter {
             }
 
             let title = Utils.localizeMessage('channel_loader.posted', 'Posted');
-            if (channel) {
+            if (channel.type === Constants.DM_CHANNEL) {
```

Comment:
We are checking `if(channel)` below but `channel.type === Constants.DM_CHANNEL` implies `channel`.  Please fix the logic here.